### ISSUE TITLE
[2/13/2017] Yahoo.finance.quotes (No definition found for Table yahoo.finance.quotes) 

### ISSUE THREAD

**Issue by bgoldstein14** [yql/yql-tables#487]
**Yahoo finance appears to have been down since around 2/9. Any news on this?**

**Comment by laurosollero** [comment]
You are passing the env argument, right? In my case I had to change it from:
`http://datatables.org/alltables.env`
to:
`store://datatables.org/alltableswithkeys`

I've found that playing with the console:
https://developer.yahoo.com/yql/console/

**Comment by bgoldstein14** [comment]
Yes, that's interesting because that variation seems to work. What I'm wondering is why suddenly the previous format you noted (which is what we're using now) suddenly stopped working!

Thank you.

**Comment by joshuakoh7** [comment]
Apologies for poking but any idea why the fix works via the YQL console but not via URL?

https://query.yahooapis.com/v1/public/yql?q=env%20%27store://datatables.org/alltableswithkeys%27;%20select%20*%20from%20yahoo.finance.quotes%20where%20symbol=%22aapl%22;
